src/modules/app_commands/sendHttpRequest.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Payload for sending http(s) request via app command.
+ *
+ * @param {string} url Http url.
+ * @param {string} serial Name the request.
+ * @param {*} [body] Parameter to the request.
+ * @param {string} [method='POST'] Http method.
+ * @param {string} [contentType='application/x-www-form-urlencoded'] ContentType header.
+ * @param {number} [timeout=1000] Request timeout.
+ *
+ * @return {AppCommand}
+ * @since lib 13.0
+ */
+export function sendHttpRequest({
+  url,
+  serial,
+  body,
+  method = 'POST',
+  contentType = 'application/x-www-form-urlencoded',
+  timeout = 10000,
+  followRedirects = false,
+  synchronizeCookies = true,
+}) {
+  const commandParameters = {
+    serial,
+    url,
+    method,
+    headers: {
+      'user-agent': navigator.userAgent,
+    },
+    contentType,
+    timeout,
+    followRedirects,
+    synchronizeCookies,
+  };
+
+  // Sending a GET request with body will kill the app.
+  // Therefore we introduce the property only if specified.
+  if (body) {
```

Comment:
The specification says that body is needed - Can we make it an optional parameter? If sending a body crashes the app we should set it optional to something the App wouldn't crash

Reply:
yeah didn't see the jsdoc - just the parameters and didn't realize that parameters are in a {}